apps/editing-toolkit/editing-toolkit-plugin/premium-content/blocks/buttons/index.js

```diff
@@ -28,6 +28,15 @@ const settings = {
 		alignWide: false,
 		lightBlockWrapper: true,
 	},
+	attributes: {
+		isPremiumContentChild: {
+			type: 'bool',
+			default: true,
+		},
+	},
+	providesContext: {
+		isPremiumContentChild: 'isPremiumContentChild',
+	},
```

Comment:
Hmm, what the heck is this?  I have no idea how this got into my PR. I've never touched this file in my life.  ??

Reply:
It's really strange, it's the exact same file on master. I thought maybe formatting issue, but the lines are new, not modified. Maybe a rebase of master would fix this.